services/catarse/app/views/catarse_bootstrap/user_notifier/mailer/project_fiscal_inform.slim

```diff
@@ -0,0 +1,92 @@
+- fiscal_data ||= @notification.fiscal_data
+- project = fiscal_data.last.project
+- user = fiscal_data.last.user
+- finished_date = fiscal_data.last.end_date.strftime('%d/%m/%Y')
+
+center
+  h1 style="line-height: 5px" INFORME DE RENDIMENTOS
+
+hr/
+
+br/
+p style="margin:0;"
+  span style="font-weight:bold;" Projeto:
+  span #{project.name}
+p style="margin:0;"
+  span style="font-weight:bold;" Beneficiário:
+  span #{user.name}
+p style="margin:0;"
+  span style="font-weight:bold;" CPF/CNPJ:
+  span #{user.cpf}
+
+
+- unless fiscal_data.sum(:total_amount_to_pf_cents).zero?
+  h3 PESSOAS FÍSICAS:
+  table style="border: 1px solid;width:100%;border-collapse: collapse;"
+    tr
+      td style="border: 1px solid;text-align:center;font-weight:bold;" INTERVALO
+      td style="border: 1px solid;text-align:center;font-weight:bold;" VALOR
+    - fiscal_data.each do |fiscal|
+      tr
+        td style="border: 1px solid;text-align:center;vertical-align: middle;white-space: nowrap;"
+          span #{fiscal.begin_date.strftime('%d/%m/%Y')} - #{fiscal.end_date.strftime('%d/%m/%Y')}
+        td style="border: 1px solid;text-align:center;vertical-align: middle;white-space: nowrap;"
+          span #{number_to_currency fiscal.total_amount_to_pf_cents, precision: 2}
+
+- unless fiscal_data.sum(:total_amount_to_pj_cents).zero?
+  h3 PESSOAS JURÍDICAS:
+  table style="border: 1px solid;width:100%;border-collapse: collapse;"
+    tr
+      td style="border: 1px solid;text-align:center;font-weight:bold;" INTERVALO
+      td style="border: 1px solid;text-align:center;font-weight:bold;" VALOR
+    - fiscal_data.each do |fiscal|
+      tr
+        td style="border: 1px solid;text-align:center;vertical-align: middle;white-space: nowrap;"
+          span #{fiscal.begin_date.strftime('%d/%m/%Y')} - #{fiscal.end_date.strftime('%d/%m/%Y')}
+        td style="border: 1px solid;text-align:center;vertical-align: middle;white-space: nowrap;"
+          span #{number_to_currency fiscal.total_amount_to_pj_cents, precision: 2}
+
+h3 DISCRIMINAÇÃO DOS RENDIMENTOS:
+
+table style="border: 1px solid;width:100%;border-collapse: collapse;"
+  tr
+    td style="border: 1px solid;text-align:center;font-weight:bold;" Valor Arrecadado
+    td style="border: 1px solid;text-align:center;font-weight:bold;"
+      span #{number_to_currency fiscal_data.sum(:total_amount_to_pf_cents) + fiscal_data.sum(:total_amount_to_pj_cents), precision: 2}
```

Comment:
**Sugestão**
Quando tu usa um `.sum` numa relation, ele dispara uma query no banco de dados, isso é bom pois caso tenha inúmeros registros, ele não carrega tudo na memória. Nesse caso específico isso não é bom, pois tu tá fazndo inúmeros `.sum` nesse arquivo de objetos que já estão carregados na memória por conta do `.each` que tu fez lá em cima, e não tem problema isso pois são poucos objetos. 
Então aqui o ideal é tu fazer um: 
```ruby
project_fiscals.to_a.sum { |pf| pf.total_amount_to_pf + pf.total_amount_to_pj }.format
```
Daí tu não fez nenhuma query, aproveitou os objetos carregados na memória e ainda está utilizando o campo monetizado ao invés dos centavos.

A mesma sugestão se aplica pros demais `relation.sum`.